cloud/pkg/cloudhub/servers/httpserver/server.go

```diff
@@ -45,6 +45,7 @@ func StartHTTPServer() {
 	router := mux.NewRouter()
 	router.HandleFunc("/edge.crt", edgeCoreClientCert).Methods("GET")
 	router.HandleFunc("/ca.crt", getCA).Methods("GET")
+	router.HandleFunc("/readz", electionHandler).Methods("GET")
```

Comment:
```suggestion
	router.HandleFunc("/readyz", electionHandler).Methods("GET")
```

Reply:
@Poor12  Please fix the suggested change and squash the commits :)

Reply:
done